min-image/runtimes/python/ol.c

```diff
@@ -8,6 +9,71 @@
 #include <sys/types.h>
 #include <sys/wait.h>
 #include <seccomp.h>
+#include <sys/capability.h>
+
+static int modify_cap_impl(int cap, int setting) {
```

Comment:
To what extent can we just create Python wrappers around the cap API (can_get_proc, cap_set_flag, etc), and then have the logic using it in Python code only?

Reply:
@tylerharter I see. Based on my research, I couldn't find any official Python libraries that would give us control over capabilities. There are some community wrappers available but I'm assuming they use similar C code underneath. I'm also concerned that if we rely on them, we could open a channel for untrustworthy libraries/ packages that aren't well maintained or supported. What are your thoughts on that?